flaggy_test.go

```diff
@@ -106,7 +106,7 @@ func TestComplexNesting(t *testing.T) {
 }
 
 func TestParsePositionalsA(t *testing.T) {
-	inputLine := []string{"-t", "-i=3", "subcommand", "-n", "testN", "-j=testJ", "positionalA", "positionalB", "--testK=testK", "--", "trailingA", "trailingB"}
```

Comment:
`-i=3` should be covered by a global flag and `--testK=testK` is a valid subcommand flag.
What was wrong with this line?